### DISCUSSION TITLE
部署在哪里访问速度会快一些

### DISCUSSION THREAD

**Discussion by ma-wenqian** [walinejs/waline#1072] [Q&A] [answered]
按照默认方式部署在Vercel，但感觉每次打开博客页面，都速度很慢，加载也很慢，请问其他的平台哪个部署访问速度会快一些？

**Comment by Mister-Hope** [comment, marked as answer]
不是敷衍你，但自己买个国内服务器效果是最快的，而且提升巨大。

waline默认是使用不同服务商的数据库和后端，且都在国外。自建环境不仅可以提高客户端到服务端的响应速度，服务端在读取本地数据库时也不再需要经历网络请求，产生延时。

**Comment by lizheming** [comment]
可以部署在 CloudBase 试一下

**Comment by z7sz** [comment]
个人建议部署在cloudbase，可以看我博客https://blog.z7sz.top
